NitroxClient/GameLogic/Entities.cs

```diff
@@ -50,23 +52,52 @@ public void Spawn(List<Entity> entities)
         {
             foreach (Entity entity in entities)
             {
+                LargeWorldStreamer.main.cellManager.UnloadBatchCells(ToInt3(entity.AbsoluteEntityCell.CellId)); // Just in case
+                EntityCell entityCell = EnsureCell(entity);
+                entityCell.Initialize();
```

Comment:
Is this safe to call again each time?  Should we call once when it is instantiated below?

Reply:
Moved into EnsureCell